src/docs/markdown/extending-caddy.md

```diff
@@ -372,4 +372,4 @@ var (
 	_ caddyhttp.MiddlewareHandler = (*Middleware)(nil)
 	_ caddyfile.Unmarshaler       = (*Middleware)(nil)
 )
-```
\ No newline at end of file
+```
```

Comment:
What is the change on this line?

Reply:
It looks like the original files doesn't have a trailing newline. (Not sure why GitHub's not rendering with the :no_entry_sign: and arrow it usually shows though)

On `master`:
```
$ xxd src/docs/markdown/extending-caddy.md | tail -1
00003dd0: 286e 696c 290a 290a 6060 60              (nil).).```
```

Reply:
Ah, yeah, weird. Usually it shows the arrow. Thanks